submission/server/prometheus.yml

```diff
@@ -0,0 +1,11 @@
+global:
+  scrape_interval:     15s
+  external_labels:
+    monitor: 'proxy-monitor'
+
+scrape_configs:
+  - job_name: 'submission-proxy'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:5951']
+      
```

Comment:
Could maybe do with a comment explaining that this is basically a copy of the sample Prometheus config file, but with job names/port set to match the defaults for the proxy.

Reply:
Added comment.